source/manuals/accessibility.html.md.erb

```diff
@@ -0,0 +1,115 @@
+
+While it may not have a full set of components for every service, it provides a solid foundation to work from that can be expanded to suit a services individual needs.
+
+## Specific guidance
+
+There is a common misconception that accessibility is difficult, time consuming and costly. This is not true, however the range of possible issues that could occur is too wide to detail here. Having said that, here are some common accessibility issues many websites have, why they’re a problem and how to avoid them.
+
+### Non semantic HTML
+
+Semantic HTML means using the appropriate tag for the element being described. A common example of this is not using a button element for a button, but relying on JavaScript to provide the correct action when the element is used.
+
+This is a problem because non-semantic HTML is difficult for technologies such as screen readers to understand, which means that some users will not be able to operate or interact with these elements, potentially rendering a service unusable.
+
+### Poor in-page navigation
+
+As websites become increasingly large it is common to have navigation sections at the top of the page containing lots of links to other parts of the site. It is important to provide a mechanism for skipping past these elements to the main content for keyboard users, such as a skip link.
+
+Areas of the page should also be built using landmarks in order to allow users to navigate more easily between them. For example, an email application might have one landmark for the email folder pane (inbox, drafts, etc.) and another for the email list pane.
+
+### Improper heading structure
+
+Screen reader users can rely heavily on correct heading structure within a page in order to navigate and understand its content. The page title should be presented within an H1 tag, and all other headings should follow a logical ordering e.g. H1, H2, H3 not H1, H3, H2.
+
+### Form controls not associated with a label
+
+All form controls should have a label associated with them that describes what the form control should be used for. [Placeholder text](https://w3c.github.io/html/sec-forms.html#the-placeholder-attribute) is not an acceptable alternative.
+
+Labels should provide a short but clear explanation of a form control. Further detail can be provided using a separate element. This should also be associated with the form control by using an attribute such as aria-describedby.
+
+### Meaningless or duplicated link text
+
+The text of a link should describe what that link points to. It should not rely on the surrounding text for context. Screen reader users can navigate pages by element type, which means that link text such as ‘click here’ or ‘read more’ does not help a user understand where the link goes.
```

Comment:
Whilst 'assistive technology' is more verbose it's a bit more true. Tools such as Dragon Naturally Speaking aren't screen readers but they can show a list of all the links on a page.

```suggestion
The text of a link should describe what that link points to. It should not rely on the surrounding text for context. Assistive technology users can navigate pages by element type, which means that link text such as ‘click here’ or ‘read more’ does not help a user understand where the link goes.
```